skimage/filters/thresholding.py

```diff
@@ -876,7 +873,7 @@ def find_local_maxima_idx(hist):
     elif counter == max_num_iter - 1:
         raise RuntimeError('Maximum iteration reached for histogram' 'smoothing')
 
-    # Find lowest point between the maxima
+    # Find the lowest point between the maxima
```

Comment:
```suggestion
    # Find the lowest point among the maxima
```

Reply:
I agree with @soupault: this is taking two maxima in the histogram and finding the minimum point between them. It is not (in general) one of the maxima.